### DISCUSSION TITLE
Has anyone mined a block with a plot created with this? (This is NOT a FUD post)

### DISCUSSION THREAD

**Discussion by number435398** [madMAx43v3r/chia-plotter#726] [Q&A]
I'm just looking for the most basic confirmation that this works to the end. This is NOT a FUD post.  I am greatly appreciative of what maxMAx43v3r has done and made available to us. But, the most complicated things can have the most minor incompatibilities and errors that may only show themselves at the very end of a given process. Has anyone actually mined a block with a plot created by this plotter? I've been inspired to spend literally thousands of dollars under the premise that this plotter works and I have not had a single plot successfully mine a single XCH yet. Maybe I'm just looking for a little encouragement.

**Comment by wabragg** [comment]
I have not.  I've only run the 'chia plots check' and received confirmation that they pass the validation tests.  I only have 345 plots though.

**Reply by number435398** [reply]
> 
> 
> I have not. I've only run the 'chia plots check' and received confirmation that they pass the validation tests. I only have 345 plots though.

I have nearly 2,000.

**Reply by wabragg** [reply]
Have you run 'plots check' yet?  It takes a while with -n 30 (default) but you can probably speed it up with -n 5 or something.  With -n 30 it takes about 25 seconds per plot check on my system.  It would take 1/2 a day to check your 2000 by that measure.  If you have any invalid plots, it will give you a summary of them at the end of the test.  I just SSH over to my machine running Chia in Docker, widen the window to prevent the text wrapping, then check back at the end when it's done.  My first run found 2 invalid which were likely my fault for trusting the output after the program faulted.  I've had no other bad ones since.

**Reply by number435398** [reply]
I have not run a plots check. I don't even know how to actually do it.  In addition to that, I'm not so much concerned about a "plots check". I'm merely asking about the very end of the process.  What if, by some strange weird contingency no one would expect, the plots pass this check but don't actually mine any coins?

**Reply by MadCoderOne332** [reply]
I am way overdue... before switching to MadMax coins were delivered on a (semi) reliable schedule for the number of plots I had. I am chalking it up to bad luck because the dev team themselves have said they are valid plots

**Reply by jayhohoho2019** [reply]
what makes the chia plots check process determine a plot is valid or invalid?  It outputs a message "Proofs N/30" and a decimal number equal to N/30.   So is there a range that makes a plot "valid?"

**Reply by wabragg** [reply]
I can now verify that I received 2 XCH overnight using plots from this plotter.

**Reply by number435398** [reply]
> 
> 
> I can now verify that I received 2 XCH overnight using plots from this plotter.

DId you have plots mixed from the original plotter too? Are you sure it was one of the plots from this plotter that was successful?

**Reply by wabragg** [reply]
I lost my original plots in the failure of a JBOD a while back and in frustration I sought out a faster plotter.  Hence, these are all from the MadMax43v3r plotter.

**Reply by number435398** [reply]
> 
> 
> I lost my original plots in the failure of a JBOD a while back and in frustration I sought out a faster plotter. Hence, these are all from the MadMax43v3r plotter.

Good to know. But you said you have only 345 and successfully mined a block? That is lucky

**Reply by wabragg** [reply]
Agreed ... lucky.  I was not expecting to wake up to 2 Chia this morning.  The farm summary tells me my average should be a block every 6 months and 2 weeks.  I ditched the JBOD on my second go for individual volumes.  And truth be told, my JBOD failure was me pulling the wrong drive out of my NAS when working on something else.

root@Scylla:# docker exec -it Chia venv/bin/chia farm summary
Farming status: Farming
Total chia farmed: 2.0
User transaction fees: 0.0
Block rewards: 2.0
Last height farmed: 532949
Plot count: 345
Total size of plots: 34.146 TiB
Estimated network space: 30843.737 PiB
Expected time to win: 6 months and 2 weeks
Note: log into your key using 'chia wallet show' to see rewards for each key
root@Scylla:#

I'm farming over on a Synology NAS using a Docker image.  The plots were generated on my workstation however.

**Reply by number435398** [reply]
My farming is a mess. I have 3 different computers farming, all saying they have like 3-4 months to win. Word of caution: Raspberry Pi 4 B can't handle more than 32 USB devices minus hubs minus ports on the back of the unit, minus (I think) the ethernet port and possibly a few other things.  Absolutely terrible for Chia mining.

**Reply by wabragg** [reply]
Should that actually be setup as a single Farm node talking to 3 Harvester nodes? I have no idea how one would do that but that's how I understand it from the documents I've seen.

I'm still trying to figure out why the Farm node runs 8 "Full_Node" processes and takes up so much memory.  I think they've still got a lot of work to do on the software.

**Reply by number435398** [reply]
My oddities of my setup will be resolved when I get some additional parts to consolidate everything into one computer.

**Reply by whitetechnologies** [reply]
They certainly pass partial proofs. On official pool protocol, all the plots have been with madmax. Pool is recognizing the entire plot count and no invalid partials have been found. That is as real world as it gets to verify this works

**Comment by alexwwstx** [comment]
I have around 1500 plots, and got nothing yet, I checked them by 'chia plots check' and looks like all are fine, just need more patience since the current netspace is so big and difficulty is 2200+ now?

**Comment by racergoodwin** [comment]
Looking for some confirmation myself, same OP.
At the moment madMAx43v3r hasn't won anything for me. I have currently 2262 madMAx43v3r plots and increasing by 100 per day with lots of passed filters, but no XCH farmed.
I am comparing against 1647 plots from the chia plotter which passes less filters, but have won 3 times (6 XCH)
Obviously luck and time play a major role, so there is still hope :)

**Reply by alexwwstx** [reply]
@racergoodwin: Since your plots number is pretty big, do you mind sharing how many days and time frame you had for those by chia plotter vs by madMAx43v3r?

**Reply by wabragg** [reply]
I'm thinking you had the original plots first when the netspace was also smaller.

**Reply by racergoodwin** [reply]
Sure.
My first plot was created on the 26th April, obviously before madMAx43v3r. Started off very slow creation time, but upgraded hardware to the point I am creating ~30 plots a day.
When I found out about madMAx43v3r, I started a separate farm (and have kept it separate) because I'm not sure about it. First plot was created on 11th June. Started off with one machine creating ~33 plots per day then added two more. Now making ~100 plots per day.

**Reply by racergoodwin** [reply]
Well, I can answer the OP question in the positive now!
What are the chances?! While writing my above reply, I went through my disks to find the earliest plots for both the original and the madMAx43v3r. Hit the reply button and went back to the farms machines to close down all the windows, and checking the madMAx43v3r farm, it had won 2 XCH!
I am including a screenshot so you can see the height they were farmed at. Current height is 536,307 as I type this. Won at 536,222. Times you can see in the screenshot are BST.

![madMAx43v3r-screenshot-2021-07-07](https://user-images.githubusercontent.com/84841970/124695849-e9675700-dedb-11eb-9468-60c7e9c7d326.png)

**Reply by alexwwstx** [reply]
nice to hear that. Thanks for your updating.

**Reply by ESP4Ever** [reply]
Hi
Re-plotting to NFT now and already joined the pool - it accepts the MaMa43 :)  shares

**Reply by number435398** [reply]
> 
> 
> Hi
> Re-plotting to NFT now and already joined the pool - it accepts the MaMa43 :) shares

What pool did you join?

**Reply by ESP4Ever** [reply]
Can't tell because it can be treated as an ad

**Reply by number435398** [reply]
> 
> 
> Can't tell because it can be treated as an ad

That's, wait, wtf? By who?  I joined a pool and, unless the UI is bugged, it seemed to indicate no one was there.  Which I guess could also be true.

**Reply by ESP4Ever** [reply]
The pool I joined has already >20k farmers and the pool space almost 120 PiB

**Comment by number435398** [comment]
I just found a block, but I can't figure out how to see which plot made it.

**Reply by wabragg** [reply]
You can find it in the logs.  It's quite a bit of junk to weed through, but you'll be looking for the items from the "harvester".  That module reports attempts and proofs found.

**Reply by number435398** [reply]
> 
> 
> You can find it in the logs. It's quite a bit of junk to weed through, but you'll be looking for the items from the "harvester". That module reports attempts and proofs found.

I can't find the location of that file. All I can find is the debug.log

**Reply by wabragg** [reply]
That's the one.   If you've set your log level to INFO, it will be pretty large.  There may be several as it rotates the logs.

**Reply by number435398** [reply]
I looked through my debug.log file for the time that I got the block, but I can't find anything referencing the finding of the block.

**Reply by wabragg** [reply]
Go into the config file "mainnet/config/config.yaml" and update the "log_level" setting to "INFO".  Right now you're probably only logging errors and warnings.

**Reply by number435398** [reply]
So I take it there's no feasible way to go back and find out which plot was used for my mined block since I didn't have that on?

**Reply by number435398** [reply]
Just had another block solve. So I went through and did all that and I could find no useful information in the log file to help me determine which plot found the block.  Am I just failing to make sense of the jargon? Have you actually been able to definitively tell which plot found a block before?

**Reply by wabragg** [reply]
My logs have rolled since I found a solution.  I believe it was in a harvester log record though.

**Reply by number435398** [reply]
I found a reference to it in the logs (with the 'INFO' Setting) and though it appeared to be a lot of information, it didn't appear to direct me to a specific plot/plot file.

**Comment by jayhohoho2019** [comment]
Is there any CLI command that shows the # of plotnft plots (in the pool)?
I understand the GUI Pool tab shows it.

**Reply by wabragg** [reply]
I have no plotnfts but I suspect it would be shown by the 'chia farm summary' command.

**Reply by alexwwstx** [reply]
cannot distinguish which ones are for nft for version 1.2.1

**Comment by gryan315** [comment]
This is an old post, but thought I'd chime in that I found a block with a madmax plot. I used a command to pull the plot filename using the plot public key found in the blockchain explorer:
```
curl --insecure --cert ~/.chia/mainnet/config/ssl/harvester/private_harvester.crt --key ~/.chia/mainnet/config/ssl/harvester/private_harvester.key -d '{}' -H "Content-Type: application/json" -X POST https://localhost:8560/get_plots | python3 -m json.tool | egrep "{|filename|plot_public_key" | fgrep -C 2 "<plot_public_key goes here>"
```
This returns the plot filename after it as well, and I was too lazy to tune it to not do that since it works. The plot happened to be one that I created just a couple days before the official pool protocol launched.

**Comment by number435398** [comment]
I should admit that since posting this, I have found several blocks, presumably with plots from this plotter. I didn't go through the ridiculously arduous process of confirming it, but considering such a large portion of my plots are from the madmax plotter, it's not reasonable to assume that they came from the plots I plotted before I found this wondrous plotter. Statistically, I can confirm that I, too, have successfully found multiple blocks.

**Comment by ghost** [comment]
I found a block with 6 TiB of NFT plots on SpacePool. Lol.

**Comment by silence48** [comment]
I found one with it about 3 days after i started plotting to the pool. meanwhile i have 2k solo plots that haven't got a block in a month and a half lol.

**Comment by svennand** [comment]
Yes. Found a block (mining pool) with my 100TB 850isj NFT plots made with madmax plotter :) after 22 days of mining

**Comment by MatejBajzeljGreyMatters** [comment]
Same. Got 3 blocks so far (1.3PB) with plots created by MadMax. This are Pool NFT plots.
